contrib/packs/actions/restart_component.yaml

```diff
@@ -10,7 +10,7 @@
       immutable: true
     cmd:
       immutable: true
-      default: "st2ctl restart-component {{servicename}}"
+      default: "service {{servicename}} restart"
```

Comment:
@DoriftoShoes , @armab, @lakshmi-kannan  guys!?

What about this 
![](https://i.gyazo.com/89ed9e06133dc7675a3812daf61dd2dd.png)

@armab check this also please https://github.com/StackStorm/st2/blame/master/st2common/bin/st2ctl#L58-L68